website/content/commands/intention/index.mdx

```diff
@@ -52,6 +52,7 @@ $ consul intention create web db
 Create an intention to deny "db" from initiating connections to _any_ service:
 
 ```shell-session
+# Wrap "*" in quotes to prevent wildcard expansion by the shell
 $ consul intention create -deny db '*'
```

Comment:
The command uses single quotes but should we be consistent with single vs double?

Reply:
I think I'll stick to double-quotes. "*" shouldn't be interpreted by the shell AFAIK, so no need for single quote specifically